app/views/users/index.html.erb

```diff
@@ -1 +1,14 @@
-<h1>Listing Users</h1>
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
@HAMMAS-SALEEM  Thanks for the suggestion.